bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/i18n/androidtv.properties

```diff
@@ -0,0 +1,73 @@
+thing-type.config.androidtv.googletv.keystorePassword.label = Keystore Password
+thing-type.config.androidtv.googletv.keystorePassword.description = Password for the keystore file
+thing-type.config.androidtv.googletv.port.label = Port
+thing-type.config.androidtv.googletv.port.description = Port to connect to
+thing-type.config.androidtv.googletv.reconnect.label = Reconnect Delay
+thing-type.config.androidtv.googletv.reconnect.description = Delay between reconnection attempts
+thing-type.config.androidtv.shieldtv.delay.label = Delay
+thing-type.config.androidtv.shieldtv.delay.description = Delay between messages
+thing-type.config.androidtv.shieldtv.heartbeat.label = Hearbeat Frequency
+thing-type.config.androidtv.shieldtv.heartbeat.description = Frequency of heartbeats
+thing-type.config.androidtv.shieldtv.ipAddress.label = Hostname
+thing-type.config.androidtv.shieldtv.ipAddress.description = Hostname or IP address of the device
+thing-type.config.androidtv.shieldtv.keystoreFileName.label = Keystore File Name
+thing-type.config.androidtv.shieldtv.keystoreFileName.description = Java keystore containing key and certs
+thing-type.config.androidtv.shieldtv.keystorePassword.label = Keystore Password
+thing-type.config.androidtv.shieldtv.keystorePassword.description = Password for the keystore file
+thing-type.config.androidtv.shieldtv.port.label = Port
+thing-type.config.androidtv.shieldtv.port.description = Port to connect to
+thing-type.config.androidtv.shieldtv.reconnect.label = Reconnect Delay
+thing-type.config.androidtv.shieldtv.reconnect.description = Delay between reconnection attempts
+
+# channel types
+
+channel-type.androidtv.app.label = App
+channel-type.androidtv.app.description = App Control
+channel-type.androidtv.appname.label = App Name
+channel-type.androidtv.appname.description = App Name
+channel-type.androidtv.appurl.label = App URL
+channel-type.androidtv.appurl.description = App URL
+channel-type.androidtv.debug.label = DEBUG Command
+channel-type.androidtv.debug.description = Binding control (for debugging)
+channel-type.androidtv.keyboard.label = Keyboard
+channel-type.androidtv.keyboard.description = Keyboard Entry
+channel-type.androidtv.keycode.label = Keycode
+channel-type.androidtv.keycode.description = Send keycode
+channel-type.androidtv.keypress.label = Key Press
+channel-type.androidtv.keypress.description = Send key press
+channel-type.androidtv.pincode.label = Pin Code
+channel-type.androidtv.pincode.description = Send Pin Code
+
+# add-on
+
+addon.androidtv.name = AndroidTV Binding
+addon.androidtv.description = This is the addon for AndroidTV.
+
+# channel types
+
+channel-type.androidtv.raw.label = RAW Command
+channel-type.androidtv.raw.description = RAW command to send (for debugging)
+channel-type.androidtv.rawmsg.label = RAW Message
+channel-type.androidtv.rawmsg.description = RAW message to handle (for debugging)
```

Comment:
Are these still relevant, or why are they at the bottom and not with the others?